src/style.css

```diff
@@ -0,0 +1,113 @@
+body {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  width: 160px;
+  height: 356px;
+  padding: 32px 16px 16px;
+}
```

Comment:
instead of fixed `height` + `space-between`, better is just to set margins for each element, so that they can later be changed